code/HISPANIA/modules/reagents/chemistry/reagents/alcohol.dm

```diff
@@ -102,3 +102,27 @@
 	drink_name = "hotline bling"
 	drink_desc = "You see Drake dancing inside"
 	taste_description ="good days in california"
+
+
+/datum/reagent/consumable/ethanol/peach_bellini
+	name= "Peach Bellini"
+	id = "peach_bellini"
+	description = "Oh la la fancy man."
+	color = "#E3E04B"
+	alcohol_perc = 0.4
+	drink_icon = "peach_bellini"
+	drink_name = "Peach Bellini"
+	drink_desc = "Oh la la fancy man."
+	taste_description ="fresh"
+
+
+/datum/reagent/consumable/ethanol/vampire_bf
+	name= "Vampire Bestfriend"
+	id = "vampire_bf"
+	description = "Do you steal the chaplain flask, son?"
```

Comment:
CIPOTE NO PUEDE SER ES "Are you stealing the chaplain flask, son?" NOOOOOOOOOOOOO

Reply:
No captastela referencia saga :cry:

Reply:
Claramente tu no captastes tu propia referencia, tal cual un mono tirando caca sin saber que hace referencias al capitulo 49 de jimmi neutron.

Reply:
nOoO saAAGa

Reply:
Es el capitulo 50

Reply:
chale me umillo :pensive: